CONTRIBUTING.md

```diff
@@ -121,23 +143,64 @@ echo "$INGRESS_HOST"
 curl -v "http://$INGRESS_HOST"
 ```
 
-1. To create monitoring examples in GCP, navigate to the monitoring folder and run 
+1. To create monitoring examples in GCP, navigate to the monitoring folder and run
 the `terraform apply` command.
 
-   2. Please note that in order to do this you will need the external IP address, project ID, 
-   and an email address. The project ID can be found in GCP or with the command `gcloud config get-value project`
+2. Please note that in order to do this you will need the external IP address, project ID,
+and an email address. The project ID can be found in GCP or with the command `gcloud config get-value project`
 
 ```bash
 cd ./monitoring
-
 terraform apply
 ```
+
 > **Troubleshooting:** A Kubernetes bug (will be fixed in 1.12) combined with
 > a Skaffold [bug](https://github.com/GoogleContainerTools/skaffold/issues/887)
 > causes the load balancer to not work, even after getting an IP address. If you
 > are seeing this, run `kubectl get service frontend-external -o=yaml | kubectl apply -f-`
 > to trigger load-balancer reconfiguration.
 
+#### ![app engine](docs/img/app-engine.png) Run App Engine micro-service on Google App Engine (GAE)
+
+> ℹ️   This task requires a running PostgreSQL DB instance with the configured rating database schema.
+> ℹ️   This task requires having App Engine application created in the project.
+
+1. Create `app.yaml` file in the `src/ratingservice` folder with the following content:
+
+```yaml
+runtime: python38
+env: standard
+service: ratingservice
+version: prod
+entrypoint: uwsgi --http-socket :8080 --wsgi-file main.py --callable app --master --processes 1 --threads 10
+env_variables:
+    DB_HOST: '[Place database hostname or IP]'
+    DB_NAME: '[Place database name]'
+    DB_USERNAME: '[Place username credential]'
+    DB_PASSWORD: '[Place password credential]'
+    MAX_DB_CONNECTIONS: 10
+basic_scaling:
+  max_instances: 10
+  idle_timeout: 10m
```

Comment:
Is there a reason why this file is not checked in with placeholders for DB password, etc? Is terraform creating it?

Reply:
Terraform module deploys without producing YAML file. The file is required only when you use Cloud SDK